sympy/polys/ring_series.py

```diff
@@ -477,32 +468,30 @@ def rs_series_inversion(p, x, prec):
     """
     R = p.ring
     zm = R.zero_monom
-    ii = R.gens.index(x)
-    m = min(p, key=lambda k: k[ii])[ii]
+    index = R.gens.index(x)
+    m = min(p, key=lambda k: k[index])[index]
     if m:
-        p = mul_xin(p, ii, -m)
+        p = mul_xin(p, index, -m)
         prec = prec + m
     if zm not in p:
-        raise NotImplementedError('no constant term in series')
+        raise NotImplementedError("No constant term in series")
 
     if _has_constant_term(p - p[zm], x):
-        raise NotImplementedError('p - p[0] must not have a constant term in the series variables')
+        raise NotImplementedError("p - p[0] must not have a constant term in "
+                                  "the series variables")
     r = _series_inversion1(p, x, prec)
     if m != 0:
-        r = mul_xin(r, ii, -m)
+        r = mul_xin(r, index, -m)
     return r
 
-
 def _coefficient_t(p, t):
-    """
-    Coefficient of `x_i**j` in p, where t = (i, j)
-    """
+    """Coefficient of `x_i**j` in p, where t = (i, j)"""
     i, j = t
-    ring = p.ring
-    expv1 = [0]*ring.ngens
+    R = p.ring
```

Comment:
Why did you rename `ring` to `R`?
Not that I'm against renaming away from `ring`, but `R` now tells the reader less about what's going on than before.